helper-functions/urls.js

```diff
@@ -0,0 +1,15 @@
+/**
+ *
+ * @param {string} rdsId
+ */
+const getImgURL = (rdsId) =>
+  `https://raw.githubusercontent.com/Real-Dev-Squad/website-static/main/members/${rdsId}/img.png`;
+
+/**
+ *
+ * @param {string} rdsId
+ */
+const getDataURL = (rdsId) =>
+  `https://raw.githubusercontent.com/Real-Dev-Squad/website-static/main/members/${rdsId}/data.json`;
+
```

Comment:
Let's move `https://raw.githubusercontent.com/Real-Dev-Squad/website-static/main/members` to `baseURL` constant

Reply:
Moved to baseURL constant.